csrc/python_frontend/fusion_definition.cpp

```diff
@@ -345,10 +345,12 @@ std::vector<at::Tensor> FusionDefinition::execute(
         // Manual schedule
         if (!user_sched.executor->isCompiled()) {
           user_sched.executor->compileFusion(
-              user_sched.scheduled_fusion.get(),
-              inputs,
-              user_sched.fusion_id_,
-              user_sched.device_id_);
+              user_sched.scheduled_fusion.get(), inputs
```

Comment:
Need Ryan's advice here.

Reply:
@rdspring1 another place I could use your help, please see the comment below.

Reply:
@rdspring1 Could you take a look here?